examples/experimental/i3s-17-and-debug/tile-layer/tile-layer.js

```diff
@@ -35,6 +35,11 @@ export default class TileLayer extends Tile3DLayer {
     if (!timeline || !viewports || !tileset3d) {
       return;
     }
+
+    if (!this.internalState.component.props.loadTiles) {
```

Comment:
Why are we putting this on internalState? that is generally deck.gl's own state.

Reply:
Because we need actual state. This method is a callback. Sometimes when it is called `this.state` contains old props.